src/views/Activities/Activities.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { Container, Header, Item, Icon, Button, Label } from 'semantic-ui-react';
+
```

Comment:
Add a comment TO-DO for the details page per activity and pagination to indicate your technical debts.

Reply:
For example, add this at the start of this code:
`// TO-DO: Details page and Pagination`

Reply:
Noted! Thank you so much! :+1: